partiql-ast/src/main/java/org/partiql/ast/v1/expr/ExprNullIf.java

```diff
@@ -0,0 +1,39 @@
+package org.partiql.ast.v1.expr;
+
+import org.jetbrains.annotations.NotNull;
+import org.partiql.ast.v1.AstNode;
+import org.partiql.ast.v1.AstVisitor;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * TODO docs, equals, hashcode
+ */
+public class ExprNullIf extends Expr {
+    @NotNull
+    public Expr value;
+
+    @NotNull
+    public Expr nullifier;
```

Comment:
You don't need to update this, but do you know the history of the name `nullifier`? I've always been confused by it:
> NULLIF (V1 , V2 ) is equivalent to the following <case specification>:
> CASE WHEN V1=V2 THEN NULL ELSE V1 END

I would personally just use `lhs` and `rhs`. I might just be missing something.